modules/rosa-sharing-vpc-cluster-creation.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-cluster-creation_{context}"]
+= Creating your cluster in a shared VPC
+
+After the AWS account user who initially created the DNS domain and the IAM roles, the VPC-owning AWS account user must create a hosted space and update the trust policy on the IAM role that was created for sharing the VPC.
+
+.Procedure
+
+. The AWS account user who owns the VPC must update the VPC sharing IAM role and add the Installer and Ingress Operator roles to the principal section of the trust policy.
```

Comment:
@EricPonvelle As this is very important configure steps for Shared-VPC install, can we make it more clear in the doc?

@oriAdler WDYT?

Reply:
@yunjiang29 @oriAdler Can you elaborate on what we need to make clear? These steps happen in the AWS console, I believe, so there's not a lot I can document from the Red Hat side. 

I could show an updated Trust Policy if that would help.

Reply:
I think an example can be great, it's a prerequisite for installation, otherwise, the backend preflight will fail, and the cluster creator will have to communicate back to the VPC owner the roles' ARNs.